frontend/src/features/admin-form/settings/SettingsEmailNotificationsPage.tsx

```diff
@@ -0,0 +1,44 @@
+import { Skeleton, Stack, Wrap } from '@chakra-ui/react'
+
+import { CategoryHeader } from './components/CategoryHeader'
+import { EmailFormSection } from './components/EmailFormSection'
+import {
+  isEmailMode,
+  isEmailOrStorageMode,
+  useAdminFormSettings,
+} from './queries'
+
+export const EmailSection = (): JSX.Element => {
```

Comment:
For react components, we usually want to avoid exporting multiple components in the same file. Our convention is to use the exported component name as the filename, which you have done. I suppose you've not planned for `<EmailSection />`  to be used outside of `<SettingsEmailNotificationsPage />`?